packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -818,6 +819,16 @@ export function LineItemSublabelAndPrice( { product }: { product: ResponseCartPr
 	return <DefaultLineItemSublabel product={ product } />;
 }
 
+function GetBillingIntervalLabel( { product }: { product: ResponseCartProduct } ) {
+	const translate = useTranslate();
+	if ( product.volume > 1 ) {
+		return translate( 'billed %(total_years)s years, then annually', {
```

Comment:
We'll need to wait for this to translate before merging as it has to do with billing language.